### ISSUE TITLE
"Ready to Exit" button activity changes

### ISSUE THREAD

**Issue by ryandiks** [natedl98/csr-rh#3]
At the moment, when a user clicks on "Ready to Exit" at the end of the intake form, the system sends an SMS to our hotline manager whether the answer is yes or no.

Request is to have the system send the SMS when the Ready to Exit answer is "yes". But when the answer is "no", let's have an email sent to the hotline manager instructing them to complete the follow up form at their convenience.

Additionally, when the user has selected "yes", we would like a confirmation popup to ensure that there is a rescue that is needed.

**Comment by natedl98** [comment]
Email functionality is already built to do what you are asking, but it will need to be setup to do this specific task

Confirmation popup is a good suggestion, the easy way to do this is to just have a default form confirmation that pops up whenever the user submits a form. I would need to think about how to implement it if you guys want it only on "yes"- it'd be easier to implement if it just kicked in during all cases.

**Comment by ryandiks** [comment]
It would be a decent compromise to have the popup at the end of the form state something like "If you have chosen 'yes' for Ready to Exit, please ensure that a rescue is required." Or similar text that makes the user think once more about choosing yes.

**Comment by natedl98** [comment]
Yeah we can probably do some sort of customizable message on submit that asks for confirmation. Thing with all of this stuff is- how do we make it generalized enough so it can work for multiple use cases (besides RH), but also configurable enough to accomplish specifically what you guys want.
